File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Enter the desired height of the pyramid: \n"); //Prompt the user for the height of the pyramid
    }
    while (height < 1 || height > 8);

    for (int i = 1; i <= height; ++i)
    {
        for (int j = 1; j <= (height - i); ++j) //Print the spaces
        {
            printf(" ");
        }

        for (int k = 1; k <= i; ++k) //Print the hashes
        {
            printf("#");
        }

        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# Prompt user for pyramid height
while True:
    try:
        height = int(input("Height: "))
        if height < 1 or height > 8:
            print("Please enter a positive number between 1 and 8")
            continue
        break
    except ValueError:
        print("Please enter a number.")

# Print pyramid
for i in range(height):
    print(" " * (height - (1 + i)), end="")
    print("#" * (1 + i), end="")
    print("  ", end="")
    print("#" * (1 + i))